storybook/src/docs/typeface.docs.mdx

```diff
@@ -20,30 +20,29 @@ In this sentence, you can see how all letters and numbers look:
   sampleText="Jouw typograaf biedt mij zulke exquise schreven 012345689"
 />
 
-## Installation
+## How to use
 
 If you are working with npm, include our package `@amsterdam/design-system-assets` in your dependencies.
 Then import `@amsterdam/design-system-assets/font/index.css` at the beginning of your stylesheet.
 
 You can also host the font yourself and include it in your application.
 Request it through the form below.
 
-If it’s not possible to use Amsterdam Sans, apply the Arial font, or the generic sans-serif font of the device.
-In all cases, specify these alternatives in case downloading the font for the user fails, or while it is in progress:
+If it is impossible to use Amsterdam Sans, use the Arial font, or the device’s generic sans-serif.
```

Comment:
If we change 'font' to 'typeface', we should change it here and everywhere else as well.

Reply:
Yes, I’ll do that.

Reply:
We decided against the change to ‘typeface’.